webknossos/webknossos/annotation/annotation.py

```diff
@@ -434,6 +436,52 @@ def _write_to_zip(self, zipfile: ZipFile) -> None:
                 layer_content,
             )
 
+    def get_annotation_dataset(self) -> Dataset:
+        """Returns a streamed dataset, incorporating fallback layers and mappings."""
```

Comment:
how are mappings incorporated? this only applies to agglomerate mappings which are pinned due to volume editing, right?

Reply:
Correct. JSON mappings and merger mode are not currently supported for this annotation streaming. May be good to mention that in the comment. It may change in the future though. I do think server-side json mappings may soon be deprecated anyway, though, compare https://scm.slack.com/archives/C5AKLAV0B/p1656524453854519

Reply:
Thanks for the information, I adapted the docstring now :writing_hand: